src/rgw/rgw_admin.cc

```diff
@@ -4817,7 +4817,11 @@ int main(int argc, char **argv)
     string user_str = user_id.to_str();
     int ret = store->cls_user_get_header(user_str, &header);
     if (ret < 0) {
-      cerr << "ERROR: can't read user header: " << cpp_strerror(-ret) << std::endl;
+      if (ret == -ENOENT) { /* in case of ENOENT */
+        cerr << "User has not been initialized " << std::endl;
```

Comment:
`User has not been initialized or doesn't exist` would be more appropriate here as the same error is thrown when the user doesn't exist as well